### PR TITLE
feat: adds post setup pages

### PR DESCRIPTION
**Pull request by nicholaswma**
@martonlederer, aside from review, can you verify that the route is appropriate for these pages?
image

Also what apps should we feature here?
image

### CODE REVIEW

**Comment by martonlederer** [comment]
I think the route should be fine

File: assets/setup/pin-example.png
Comment (martonlederer) [inline assets/setup/pin-example.png:1]: This is really low quality. Can we please change it.

File: src/routes/welcome/gettingStarted.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+              <Page key={page} ref={contentRef}>
+                {gettingStartedPages[page - 1]}
+              </Page>
+            </AnimatePresence>
+          </PageWrapper>
+        </Content>
+        <Footer>
+          <PageIndicatorContainer>
+            {gettingStartedPages.map((_, i) => (
+              <PageIndicator
```
Comment (martonlederer) [inline src/routes/welcome/gettingStarted.tsx:62]: The white bg for the other page indicators looks off. Let's change that to something that is consistent with ArConnect's style. Maybe `rgba(theme_color, .3)`, we use that for secondary bgs. Like so:
Screenshot 2023-10-20 at 7 34 19 PM

File: assets/ecosystem/publish-logo.svg
Comment (martonlederer) [inline assets/ecosystem/publish-logo.svg:1]: This logo is not visible in dark mode

File: src/routes/welcome/gettingStarted/connect.tsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+  return (
+    <Wrapper>
+      <div>
+        <Text heading>{browser.i18n.getMessage("connect_with_us_title")}</Text>
+        <Paragraph>{browser.i18n.getMessage("connect_paragraph")}</Paragraph>
+        <Paragraph>{browser.i18n.getMessage("connect_paragraph_2")}</Paragraph>
+      </div>
+      <Content>
+        <Item
```
Comment (martonlederer) [inline src/routes/welcome/gettingStarted/connect.tsx:26]: Could you please add the usual animations, so the users can "feel" feedback when hovering/pressing these logos. (by usual, I mean, ~.8 opacity on hover, ~.925 scale on `:active`).
Screenshot 2023-10-20 at 7 38 54 PM

**Review by martonlederer — CHANGES_REQUESTED** [review]
I've added some comments, once those are fixed, this is good to merge. Looks good, amazing work!

File: assets/setup/pin-example.png
Comment (nicholaswma) [reply assets/setup/pin-example.png:1]: oops, moved it to an svg

File: assets/ecosystem/publish-logo.svg
Comment (nicholaswma) [reply assets/ecosystem/publish-logo.svg:1]: will be merging this into the feature branch for the time being, but we can change the featured apps to something that will fit better

**Pull request merged**